packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -326,11 +339,31 @@ export default function makeKernelKeeper(
     return ri;
   }
 
-  /**
-   * @returns { boolean }
-   */
-  function getEnableFakeDurable() {
-    return !!kvStore.get('kernel.enableFakeDurable');
+  function setDefaultReapInterval(interval) {
+    assert(
+      interval === 'never' || isNat(interval),
+      'invalid defaultReapInterval value',
+    );
+    kvStore.set('kernel.defaultReapInterval', `${interval}`);
+  }
+
+  function getNat(key) {
+    const result = Number.parseInt(getRequired(key), 10);
+    assert(isNat(result));
+    return result;
+  }
+
+  function getSnapshotInitial() {
+    return getNat('kernel.snapshotInitial');
+  }
+
+  function getSnapshotInterval() {
+    return getNat('kernel.snapshotInterval');
+  }
+
+  function setSnapshotInterval(interval) {
+    assert(isNat(interval), 'invalid snapshotInterval value');
```

Comment:
This might be more humane if it mentioned the non-Nat value in the error message.

Reply:
I deliberately didn't do that in case the non-Nat value was something that exploded when you try to stringify it.

Reply:
unfortunate but yeah

Reply:
Actually, I can do `${JSON.stringify(value)}` which is ugly but safer the `${value}`

Reply:
On second thought, no, `JSON.stringify` will blow up on bigints (whereas `${value}` will blow up on symbols).